hawkular-rest-status/src/main/java/org/hawkular/commons/rest/status/RestStatusHandler.java

```diff
@@ -35,33 +34,42 @@
 import com.wordnik.swagger.annotations.Api;
 
 /**
- * REST endpoint for status.
+ * REST endpoint for baseStatus.
  *
- * <p>This handles the {@code /status} URL and outputs information read from the META-INF/MANIFEST.MF file together
- * with whatever information injected into it by maps of strings qualified by {@link RestStatusInfo @RestStatusInfo}.
- * There can be any number of such maps (including none) and all of them will be included in the output.
+ * <p>
+ * This handles the {@code /baseStatus} URL and outputs information read from the META-INF/MANIFEST.MF file of the
+ * present servlet context together with whatever information injected into it by maps of strings qualified by
+ * {@link RestStatusInfo @RestStatusInfo}. There can be any number of such maps (including none) and all of them will be
+ * included in the output.
  *
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  * @author Lukas Krejci
  */
-@Path("/status")
-@Api(value = "/status", description = "Status of the component service.")
+@Path("/baseStatus")
```

Comment:
-1 on this, too :) The purpose of this class is to be reused by the components so that the components can report their statuses without much code duplication. Components can also add their own specific data do the output. Calling the endpoint "baseStatus" doesn't make much sense to me from that POV.